.travis.yml

```diff
@@ -56,9 +56,12 @@ install:
     - echo ${TEST}
     - echo ${PYTHON}
 
-    - if [ "${QT}" == "pyqt" ]; then
+    - if [ "${QT}" == "pyqt5" ]; then
           conda install pyqt --yes;
       fi;
+    - if [ "${QT}" == "pyqt4" ]; then
+          apt-get install python-qt4 python-qt4-gl;
```

Comment:
You can see the available installs here: https://anaconda.org/anaconda/pyqt/files.  You'll probably need to skip the py26 + pyqt5 combo as there does not appear to be a py26 package on the defaults channel for that